packages/page-staking/src/Actions/Account/ListNominees.tsx

```diff
@@ -31,7 +34,10 @@ function ListNominees ({ nominating, stashId }: Props): React.ReactElement<Props
         </Expander>
       )}
       {nomsInactive && nomsInactive.length !== 0 && (
-        <Expander summary={t<string>('Inactive nominations ({{count}})', { replace: { count: nomsInactive.length } })}>
+        <Expander
+          help={t<string>('The active validator list from which the algorithm can select for upcomming eras.')}
```

Comment:
This is the list of active that has not been selected, but may be.

Reply:
I'd like to present this from the nominator's pov. Am I right in saying that "you were not selected as a nominator" and "the algo didn't select this validator" is similar?
```suggestion
          help={t<string>('The elected validator list that did not get selected by the Phragmen algorithm for this era. However they may be selected in the future.')}
```